source/connect.txt

```diff
@@ -267,17 +250,17 @@ To disconnect from a deployment and exit ``mongosh``, you can:
 Limitations
 -----------
 
-- :manual:`Kerberos authentication </core/kerberos/>` does not permit
+- :manual:`Kerberos authentication </core/kerberos/>` does not allow
   ``authMechanismProperties=CANONICALIZE_HOST_NAME:true|false`` in the
-  connection string. Use one of:
-  ``authMechanismProperties=CANONICALIZE_HOST_NAME:forward|forwardAndReverse|none``
-  instead.
+  connection string. Instead, use either:
+  
+  - ``authMechanismProperties=CANONICALIZE_HOST_NAME:forward``
+  - ``authMechanismProperties=CANONICALIZE_HOST_NAME:forwardAndReverse``
+  - ``authMechanismProperties=CANONICALIZE_HOST_NAME:none``
 
 - ``mongosh`` currently only supports the ``zlib``
   :manual:`compressor </core/wiredtiger/#compression>`. The following 
   compressors are not supported:
 
   - ``zstd``
   - ``snappy``
-
-
```

Comment:
[suggestion] Looking at this page from an information typing perspective, I'm not sure all these tasks actually belong on this page. Something to perhaps file a ticket about and consider later.